README.md

```diff
@@ -6,3 +6,9 @@ Instructions to deploy on OCP >= 4.2 using OLM can be found [here](docs/deploy-w
 
 ## Using the must-gather image with the local storage operator
 Instructions for using the local storage's must-gather image can be found [here](docs/must-gather.md)
+
+## Bumping OCP version in OLM metadata
```

Comment:
This does not update just the metadata, but also CSV version in `local-storage-operator.package.yaml`. See my comment below for `update-metadata.sh` script.